src/components/ContentWidgets/TodoList/TodoListEditor.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+
+/**
+ * TodoListEditor creates an editable widget to change what is displayed in the TodoList
+ * 
+ * Last Modified
+ * Nitesh Chetry
+ * July 3, 2019
+ */
+
+export const TodoListEditor: React.FC<WidgetEditorProps> = ({
+    editedContent,
+    setEditedContentOnChange
+}) => {
+    const list = editedContent.todoList_items || [];
+    let newList = [...list]
+    const [textItem, changeTextItem] = useState("");
+    return <>
+        <form onSubmit={(e) => {
+            e.preventDefault();
+            if (textItem) {
```

Comment:
With strings I think you *must* do `if (textItem === "")` because falsey values are weird with strings from what I remember